gbtis/obj/Debug/gbtis_MarkupCompile.i.cache

```diff
@@ -4,17 +4,17 @@
 winexe
 C#
 .cs
-c:\users\adambatson\documents\visual studio 2015\Projects\gbtis\gbtis\obj\Debug\
+D:\repositories\gbtis\gbtis\obj\Debug\

```

Comment:
Files like this one are probably going to have to be .gitignore'd?  Or else this will always be changing anytime someone commits from their personal PC?